src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -60,24 +69,12 @@ var DOMRenderer = ReactFiberReconciler({
     children : HostChildren<Instance | TextInstance>,
     internalInstanceHandle : Object
   ) : Instance {
-    const domElement : Instance = document.createElement(type);
+    const root = document.body; // HACK
```

Comment:
Yea, I was going to keep a stack like owner is implemented and look up the top of it, then pass that into the "host config".